docs/postresql.md

```diff
@@ -0,0 +1,23 @@
+# Install and Use PostgreSQL 
+
+##Step 1 — Installing PostgreSQL
+    sudo apt-get update
+    sudo apt-get install postgresql postgresql-contrib
```

Comment:
I'm pretty confident in a year apt updates version of PostgreSQL will be changed. Think there might be a lot of problems with PostgreSQL of higher than expected version.